From a101d3898987f9677a1819c4f69db3a66491957d Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Mon, 18 Jan 2021 19:00:58 +0530
Subject: [PATCH] Fix signapk.jar path for RELEASE_BUILD=true case

This patch fixes the issue with signapk.jar path.
This path was set to the script directory but needed to be
set to the output directory.
Without this if code built with RELEASE_BUILD=true will fail.
As the cass-sign****.zip will not be generated and signapk.jar
will not be found.

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Change-Id: If14c6aabef0c8d0bb2e86f145b74d520f4173d61
---
 tools/releasetools/common.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/releasetools/common.py b/tools/releasetools/common.py
index 0172add3a4..79b3211eca 100644
--- a/tools/releasetools/common.py
+++ b/tools/releasetools/common.py
@@ -64,7 +64,8 @@ class Options(object):
           'executable -- build and run `{}` directly.'.format(
               script_name[:-3]),
           file=sys.stderr)
-    self.search_path = os.path.dirname(os.path.dirname(exec_path))
+    #self.search_path = os.path.dirname(os.path.dirname(exec_path))
+    self.search_path = os.path.dirname(os.path.join(os.path.dirname(exec_path), '../../../../out/host/linux-x86'))

     self.signapk_path = "framework/signapk.jar"  # Relative to search_path
     self.signapk_shared_library_path = "lib64"   # Relative to search_path
--
2.17.1

